modules/administration-guide/partials/con_che-server-components.adoc

```diff
@@ -0,0 +1,23 @@
+[id="{prod-id-short}-server-components_{context}"]
+= {prod-short} server components
+
+include::example$snip_using-devworkspace-warning.adoc[]
+
+.{prod-short} server components interacting with the {devworkspace} engine
+image::architecture/{project-context}-deployments-interacting-with-devworkspace.png[]
+
+The {prod-short} server components ensure multi-tenancy and easier workspaces management.
+
+// The following 2 notes should be commented out until workspaces authentication is supported:
+
+// NOTE: When {prod-short} uses the {devworkspace} engine, {identity-provider} is not a requirement anymore. On vanilla Kubernetes, {prod-short} can be configured to use any OpenID Connect (OIDC) provider (the default is https://dexidp.io/[Dex]). On OpenShift, {prod-short} uses the available OpenShift OAuth v2 service.
```

Comment:
You say, `the following two notes` but I only see one note.  Where is the other note to comment out? 

Also, I would say `The following two notes...` that is, spell out `two`.